donate/settings/__init__.py

```diff
@@ -0,0 +1,30 @@
+from .environment import env
+from .base import Base
+from .braintree import Braintree
+from .database import Database
+from .redis import Redis
+from .secure import Secure
+from .sentry import Sentry
+from .testing import Testing
+from .development import Development, ThunderbirdDevelopment
+from .review_app import ReviewApp
+from .staging import Staging, ThunderbirdStaging
+from .production import Production, ThunderbirdProduction
+
+__all__ = [
```

Comment:
Django has so much magic in it, I really don't understand why there is no way to just tell it to imexport everything in the dir...